test/serializer/optimized-functions/DeadObjectAssign21.js

```diff
@@ -1,8 +1,8 @@
-// Copies of _\$E\(:3
-// Copies of var _\$E = _\$D.assign;:1
+// Copies of _\$8\(:3
```

Comment:
@trueadm looks like we have really hard to maintain test case here.

Reply:
We need a better way of creating names for variables rather than incrementing a counter really. That's the main problem here.